common/common_structs/src/alias_types.rs

```diff
@@ -4,6 +4,7 @@ use crate::{LockedAssetTokenAttributesEx, UnlockSchedule};
 
 pub type Nonce = u64;
 pub type Epoch = u64;
+pub type Week = u64;
```

Comment:
`Week` is `usize`, not u64.

Also, please import the `energy-integration/common-types` module and use the one from there, instead of re-importing.